script.js

```diff
@@ -0,0 +1,59 @@
+const titleInput = document.querySelector('#book');
+const author = document.querySelector('#author');
+const add = document.querySelector('#add-btn');
+const bookContainer = document.querySelector('.book-container');
+
+function setValue() {
+  const currentstoreddata = localStorage.getItem('storeddata');
+  const parseddata = JSON.parse(currentstoreddata);
+
+  for (let i = 0; i < parseddata.length; i += 1) {
+    const newBook = document.createElement('p');
+    const newAuthor = document.createElement('p');
+    const removeBtn = document.createElement('button');
+    const hr = document.createElement('hr');
+
+    bookContainer.appendChild(newBook);
+    bookContainer.appendChild(newAuthor);
+    bookContainer.appendChild(removeBtn);
+    bookContainer.appendChild(hr);
+
+    newBook.innerHTML = parseddata[i].book;
+    newAuthor.innerHTML = parseddata[i].author;
+    removeBtn.innerHTML = 'Remove';
+
+    removeBtn.addEventListener('click', () => {
+      bookContainer.removeChild(newBook);
+      bookContainer.removeChild(newAuthor);
+      bookContainer.removeChild(removeBtn);
+      bookContainer.removeChild(hr);
+      parseddata.splice(i, 1);
+      localStorage.setItem('storeddata', JSON.stringify(parseddata));
+    });
+  }
+}
+
+function populateStorage() {
+  const currentstoreddata = localStorage.getItem('storeddata');
+  const bookArray = JSON.parse(currentstoreddata);
+  const bookData = {};
+  let stringifydata = '';
+  bookData.book = titleInput.value;
+  bookData.author = author.value;
+
+  titleInput.value = '';
+  author.value = '';
+  bookArray.push(bookData);
+  stringifydata = JSON.stringify(bookArray);
+  localStorage.setItem('storeddata', stringifydata);
+  bookContainer.innerHTML = '';
+  setValue();
+}
```

Comment:
- [ ] It looks like there is a problem with your function to add a book. As a result of this, I am unable to add books
 to your app . I have attached a video below for your reference :100: 
[Screencast from 02-13-2023 05:52:22 PM.webm](https://user-images.githubusercontent.com/102699958/218536308-98fdc187-fdcb-42d0-ad8a-264924284615.webm)
- [ ] It throws this error in the console